package.json

```diff
@@ -24,7 +24,8 @@
   },
   "homepage": "https://github.com/tshaddix/react-chrome-redux#readme",
   "dependencies": {
-    "lodash": "^4.5.1"
+    "lodash": "^4.5.1",
+    "redux": "^3.7.2"
```

Comment:
This was required for the typescript definition "import" statement to work. Followed example from `@types/react-redux`

Reply:
ack ack!